### DISCUSSION TITLE
Left community due to too many animal abuse images on discord.

### DISCUSSION THREAD

**Discussion by BlackMoons** [FakeFishGames/Barotrauma#14546] [Ideas and suggestions]
Leaving the community and delisting all my mods due to excessive un-moderated simulated animal abuse gifs posted on the discord. Tried bringing it to attention to moderators but apparently nobody cares.

**Comment by Ek-Videogames** [comment]
![image](https://github.com/user-attachments/assets/0aa1b3fd-95e7-478e-8755-ea6282ee2902)

Edit: I refuse to believe you have ever left your house, because you would not survive out in the world. 'Snowflake' is not in my regular vocabulary, but I daresay such an extreme case as this is the exception.
(nice botnet btw)

**Reply by BlackMoons** [reply]
And this kind of toxicity is exactly why I am leaving.

**Reply by PanmanS** [reply]
Idk how gifs you mentioned in discord were animal abuse but welcome to Internet ig.

**Comment by Dynamoon** [comment]
Hi everyone, a couple of thoughts here:

"Leaving notes" like this are generally prone to less than healthy discussions. Whatever the circumstances of choosing to leave one of our platforms, we're always sorry to see you go, but it's best to keep your reasoning to yourself & friends, and not make it a public affair.

To clarify the circumstances here as best we understand them, the "animal abuse images" cited in the topic were by and large gifs featuring kittens and explosions – done in a way that made it clear that no kittens were actually harmed, and furthermore in a way that I don't believe suggested the people posting the gifs were in any way inclined to harm actual animals. I love kittens too, and while such gifs are not in particularly good taste, calling them animal abuse is an overstatement. Depictions of the abuse of animals or other living things do not go unmoderated in our server. All of that being said, I am still sorry OP felt ignored by our moderators!

Finally, while our GitHub is a great place to reach our developers, we must reserve this space for game development related posts. Requests related to community moderation should be taken up directly with the moderators of the platform in question, or our CMs (myself and Vihis).

With that, we'll be locking the thread. Thank you for your understanding!
